mcrouter/lib/network/gen/MemcacheRouterStats.h

```diff
@@ -108,7 +112,7 @@ MemcacheRouterStatsConfig::getStatGroup<McGetRequest>() {
 template <>
 inline constexpr size_t
 MemcacheRouterStatsConfig::getStatGroup<McGetsRequest>() {
-  return 5; // stat group 'other'
+  return 6; // stat group 'other'
```

Comment:
If we decide to bundle `gets` in with `get` and `cas` in with `set, then e would not change anything above and we would just change the return value here to the corresponding index.